website/integrations/services/oracle-cloud/index.md

```diff
@@ -46,3 +46,76 @@ Create a new _Social IdP_ via the _Add IdP_ button. Set the name to authentik an
 Set the _Discovery service URL_ to `https://authentik.company/application/o/oracle-cloud/.well-known/openid-configuration` and save the IdP. The IdP has now been created but must be enabled before it can be used to login with.
 
 Navigate to _IdP Policies_ in the sidebar and open the default policy by clicking on it. Edit the first rule within the policy. Add authentik under _Assign identity providers_. Here you can optionally also remove username-based logins, however it is recommended to not remove the option until you've verified SSO works.
+
+## Alternative method: Federation using a SAML Provider from Metadata
+
+When the Orancle Cloud tenancy does not have _Identity & Security_ menu does not have the _Domains_ entry but _Federation_, then this method will have to be used instead.
+
+In contrast to the OpenID Method, you no OCI user is used or created for our the SAML logins, but the a Group Mapping is used instead.
+
+Also Policies have to be created which allow the Federated users access. By default, tehy can log in, but cannot see or do anything.
+
+Creating Policies has not been tested and is not described here.
+
+See https://docs.oracle.com/en-us/iaas/Content/Identity/Concepts/federation.htm for more information.
+
+### Federation Method, Step 1 - Oracle Cloud
+
+In Oracle Cloud, open the top-left navigation and go to _Identity & Security_ and then _Federation_.
+
+Below the list of active Identity Providers, you should see an information panel with this message:
+
+> (i) You need the Oracle Cloud Infrastructure Federation Metadata document when setting up a trust with Microsoft Active Directory Federation Services or with other SAML 2.0-compliant identity providers. This is an XML document that describes Oracle Cloud Infrastructure endpoint and certificate information. __Download this document__ or Learn more.
+
+Download the XML Document behind the link behind __Download this document__.
+
+### Federation Method, Step 2 - authentik
+
+In authentik, under _Providers_, create an _SAML Provider from Metadata_ with these settings:
+
+-   Name: Oracle Cloud SAML <your tenancy> (or whatever you like as name for this provider)
+-   Authorization flow: default-provider-authorization-explicit-consent (Authorize Application), implicit-consent may work too.
+-   Metadata: Browse to the downloaded XML Document and 
+
+Click Finish. Edit the Provider, below Protocol settings:
+
+-   ACS URL: Copy this URL
+-   Audicence: If empty, paste the ACS URL
+
+Click Update. Click on the name of the new provider in the list to see the overview. Use the link _Create Application_ on this page to create the application for the provider.
+
+Create an application which uses this provider. Optionally apply access restrictions to the application using policy bindings.
+
+-   Name: Oracle Cloud SAML <your tenancy> (or whatever you like as name for this application)
+-   Slug: oracle-cloud-saml-<your tenancy> (or whatever yoou like)
```

Comment:
```suggestion
-   Name: Oracle Cloud SAML <your tenancy>
-   Slug: oracle-cloud-saml-<your tenancy>
```

We usually try to avoid those as most users know they can input what they want.